### ISSUE TITLE
Requires double-drag when closing a backgrounded app

### ISSUE THREAD

**Issue by okmsbun** [tanujnotes/Olauncher#235]
I loved Launcher. It's very simple and perfect launcher. But when I put an application to the background to close it, I cannot close the application in one drag. I have to drag it twice each time.

I have put the screen video of the problem I encountered below.

Device: Samsung Galaxy A52s 5G (SM-A528B/DS) Android 12

https://user-images.githubusercontent.com/49310621/193328036-1fb18fb1-6cc5-4f7b-949f-231c5b1ac97e.mp4

**Comment by tanujnotes** [comment]
Most likely it's phone/OS issue because Olauncher does nothing related to your recent apps.

Can you download any other launcher and see if the issue still exists?

And even if it doesn't, I can't replicate this issue and hence can't fix it.

**Comment by okmsbun** [comment]
As you guessed, the problem is in the operating system.  There is no problem with the device's own launcher, but I faced the same problem with other launchers.

* There is no problem with the device's own launcher 'One UI Launcher'.

https://user-images.githubusercontent.com/49310621/193348331-d5e5d7df-4b8b-40d1-8ecd-7f80a465f2b6.mp4

* I faced the same problem in other launchers.
* Nova Launcher

https://user-images.githubusercontent.com/49310621/193348739-b9d302cc-8cd2-4d70-99c5-fcd0ab34cd8d.mp4

* Niagara Launcher

https://user-images.githubusercontent.com/49310621/193348921-79fd79b3-5e84-4a58-b551-1e9059ed879f.mp4

And after a short wait after throwing the application into the background in the launchers, I realized that it was working properly.  But waiting is not a good thing.  I guess I won't be using another launcher until the OS fixes this problem.  Thanks for the help.

* It fixes when you wait

https://user-images.githubusercontent.com/49310621/193350221-542ee351-619c-47cb-af1e-51b0d47bee62.mp4
